docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -20,7 +20,7 @@ The minimal required version of .NET Framework is 4.7.2.
     - Ensure Visual Studio is on Version "16.5" or greater
     - Ensure "Use previews of the .NET Core SDK" is checked in Tools -> Options -> Environment -> Preview Features
     - Restart Visual Studio
-1. [.NET Core SDK 5.0 Preview 7](https://dotnet.microsoft.com/download/dotnet-core/5.0) [Windows x64 installer](https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-5.0.100-preview.7-windows-x64-installer)
+1. [.NET Core SDK 5.0 Preview 8](https://dotnet.microsoft.com/download/dotnet-core/5.0) [Windows x64 installer](https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-5.0.100-preview.8-windows-x64-installer)
```

Comment:
Per @jaredpar's comments, this also bumps us to requiring 16.8 -- do we need to update line 18 accordingly?

Reply:
💡 If we update the solution file appropriately, earlier versions of VS will actually know to report an error if you try to open it.

https://github.com/dotnet/roslyn/blob/255cb1e0543c986960da2bfdf265842c64fcc4a5/Roslyn.sln#L4